### ISSUE TITLE
A problem about AVSD dataset

### ISSUE THREAD

**Issue by woshiawang** [kevinliang888/IVR-QA-baselines#1]
This is very good work, can you provide a link to the AVSD dataset, I can't find the dataset at the moment, the official website only has videos of the training and validation sets, not the test set. I can't find the dataset at the moment. Thank you!

**Comment by kevinliang888** [comment]
Thanks for your interest in our work. The AVSD dataset consists of 7985 training samples and 1863 validation samples. We split the validation samples into a new validation split of 863 samples and 1000 test samples. Thus, our test set is derived from the original validation samples of the AVSD dataset.
